src/plugins/d3/renderer/hooks/useSeries/prepareSeries.ts

```diff
@@ -84,6 +90,14 @@ function prepareBarXSeries(args: PrepareBarXSeriesArgs): PreparedSeries[] {
             data: singleSeries.data,
             stacking: singleSeries.stacking,
             stackId: singleSeries.stacking === 'normal' ? commonStackId : getRandomCKId(),
+            dataLabels: {
+                enabled: singleSeries.dataLabels?.enabled || false,
+                inside:
+                    typeof singleSeries.dataLabels?.inside === 'boolean'
+                        ? singleSeries.dataLabels?.inside
+                        : false,
```

Comment:
U could use lodash get or smth like `singleSeries.dataLabels?.inside ?? false`. I think it's less verbose